azure-ipam/main.go

```diff
@@ -22,9 +22,9 @@ func main() {
 func executePlugin() error {
 	// logger config
 	loggerCfg := &logger.Config{
-		Level:            "debug",
-		OutputPaths:      "var/log/azure-ipam.log",
-		ErrorOutputPaths: "var/log/azure-ipam.log",
+		Level:                "debug",
+		Filename:             "var/log/azure-ipam.log",
+		MaxSize:              10, //MB
```

Comment:
lets have 40 MB and 5 MB per file and 8 files in total

Reply:
Will address in the new commit